tests/yadism/test_output.py

```diff
@@ -25,6 +25,7 @@ def xfxQ2(self, pid, x, Q2):
 class TestOutput:
     def fake_output(self):
         out = dict()
+        out["xgrid"] = {"grid": np.array([0.5, 1.0]).tolist()}
         out["interpolation_xgrid"] = np.array([0.5, 1.0])
```

Comment:
Now there are two `xgrid`s (`out["xgrid]` and `out["interpolation_xgrid"]`) keep a single one